client-implementation/pom.xml

```diff
@@ -11,12 +14,114 @@
     <artifactId>smallrye-graphql-client</artifactId>
     <name>SmallRye: MicroProfile GraphQL Client Implementation</name>
 
-    <dependencies>        
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                    <parameters>true</parameters>
```

Comment:
Why is this needed ?

Reply:
Parameter names are used as a fallback when a parameter is not annotated as `@Name`, which I really like. Too bad Oracle hasn't made this option the default.

Reply:
Jandex should give you the name. But we can add that later.